packages/uuid/src/createUuid.ts

```diff
@@ -0,0 +1,11 @@
+import { Uuid } from './types'
+import { generateRandomNumbers } from './generateRandomNumbers'
+import { generateUuid } from './generateUuid'
+import { pipe } from '@typed/functions'
+
+/**
+ * Creates a universally unique identifier that works in both browser and Node.js
+ * environments.
+ * @name createUuid(): Uuid
+ */
+export const createUuid = pipe(generateRandomNumbers, generateUuid) as () => Uuid
```

Comment:
Suggestion:

`createUuid` -> `uuid`

I don’t see any need to have a factory name here, unless you plan to make other UUID versions.